tests/features/solution/homepage.feature

```diff
@@ -11,6 +11,9 @@ Feature: Solution homepage
     And the following contact:
       | name  | Placebo             |
       | email | Placebo@example.com |
+    And the following licence:
+      | title       | LGPL                                |
```

Comment:
Why this was added? It seems that is not used anywhere in the test, only in the data.